### PR TITLE
docs: Update showcase

### PR DESCRIPTION
**Pull request by wittjosiah**
Currently dev server version works but bundle fails

### CODE REVIEW

**Comment by wittjosiah** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2172** <a href="https://app.graphite.dev/github/pr/dxos/dxos/2172" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dxos/dxos/2172?utm_source=stack-comment).

File: docs/docs/guide/quick-start.md [hunk 1]
```diff
@@ -14,6 +14,20 @@ DXOS is the developer platform for **collaborative**, **offline-first**, **priva
 *   [Installing KUBE infrastructure](#starting-a-kube) for hosting apps locally
 *   [Deploying apps to KUBE](#deploying-your-app-to-a-kube)
 
+:::showcase[demo=Test#L5-L13]
```
Comment (zhenyasav) [inline docs/docs/guide/quick-start.md:17]: i dont think we mean to leave this here right?

**Review by zhenyasav — CHANGES_REQUESTED** [review]

File: docs/docs/guide/quick-start.md (hunk 1 above)
Comment (wittjosiah) [reply docs/docs/guide/quick-start.md:17]: Correct, this is just a demo for testing purposes. Once this is fully working I'll work with you to get one in a proper stop.

**Comment by wittjosiah** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: kFalrtK07bVMgcGAcZYV.
See details on [graphite.dev](https://app.graphite.dev/github/pr/dxos/dxos/2172)

**Pull request merged**